.github/workflows/build-and-deploy-notifications.yml

```diff
@@ -0,0 +1,65 @@
+name: Build and Deploy
+
+on:
+  push:
+    branches:
+    - main
+    - deploy-resources
```

Comment:
We only needed to trigger on `deploy-resources` for testing, this can be removed